mithril-core/src/error.rs

```diff
@@ -102,6 +102,18 @@ pub enum MerkleTreeError<D: Digest> {
 /// Errors which can be outputted by key registration.
 #[derive(Debug, Clone, thiserror::Error, PartialEq, Eq)]
 pub enum RegisterError {
+    /// This key is not available in the cardano_stake distribution
+    #[error("Key not available in Cardano stake distribution.")]
+    KeyNonExisting,
+
+    /// Invalid Operational Certificate
+    #[error("The submitted Operational Certificate is Invalid")]
+    InvalidOpCert,
+
+    /// Invalid KES signature of Mithril key
+    #[error("The submitted KES signature is invalid")]
+    KesSignatureInvalid,
```

Comment:
@iquerejeta, these errors should be in `mithril-common` to keep `mithril-core` agnostic as viewed during our pairing session